src/components/CommentList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Comment } from './Comment';
+import { CommentTypes } from '../types';
+
+export const CommentList = ({ comments }) => (
+  <div className="posts__item-comments">
+    <p className="posts__item-comments-amount">
+      {`Comments: ${comments.length}`}
+    </p>
+    <div className="posts__item-comments-list">
+      {comments.map(comment => (
+        <div className="posts__item-comments-item" key={comment.id}>
+          <Comment {...comment} />
+        </div>
```

Comment:
move this div inside Comment

```suggestion
          <Comment {...comment}  key={comment.id} />
```